setup.cfg

```diff
@@ -49,6 +49,9 @@ install_requires=
 [options.extras_require]
 tests =
     pytest
+    pytest-cov
+    pytest-xdist
+    coverage != 6.3.0
```

Comment:
Are these additions required to switch to ruff? Or are they part of that commit to be reverted you mentioned? 

If they aren't required to run ruff you should update the pull request to describe the reason they are added.

Reply:
Those are moved from `.github/workflows/ci.yml`, that the CI setup does not differ from local installs.